clients/venice-admin-tool/src/main/java/com/linkedin/venice/datarecovery/StoreRepushCommand.java

```diff
@@ -59,7 +64,7 @@ private List<String> generateRepushCommand() {
     cmd.add(this.params.command);
     cmd.add(this.params.extraCommandArgs);
     cmd.add(String.format("--store '%s'", this.params.store));
-    cmd.add(String.format("--fabric '%s'", this.params.sourceFabric));
+    cmd.add(String.format("--fabric '%s'", this.params.destFabric));
```

Comment:
This is not right, we should still use the sourceFabric for this command, it specifies where we copy data from